lib/ts/controllers/s-expandable-control.ts

```diff
@@ -69,8 +69,8 @@
         // for non-checkable elements, the initial source of truth is the collapsed/expanded
         // state of the controlled element (unless the element doesn't exist)
         _isCollapsedForClickable() {
-            var cc = this.controlledCollapsible;
-            return cc ? !cc.classList.contains("is-expanded") : this.element.getAttribute("aria-expanded") === "false";
+            var cc = this.controlledCollapsibles;
+            return cc.length > 0 ? !cc.every(element => element.classList.contains("is-expanded")) : this.element.getAttribute("aria-expanded") === "false";
```

Comment:
nit: adding comment for myself here. Commit as desired

```suggestion
            // the element is considered collapsed if *any* target element is collapsed
            return cc.length > 0 ? !cc.every(element => element.classList.contains("is-expanded")) : this.element.getAttribute("aria-expanded") === "false";
```